balrogscript/docker.d/worker.yml

```diff
@@ -69,3 +69,21 @@ server_config:
       - 'esr'
       - 'esr-localtest'
       - 'esr-cdntest'
+  staging:
```

Comment:
Hm, how is this different from `dep` ? Other than a more intuitive name? :)

Reply:
It matches the xpi scope in [grants.yml#l1813](https://hg.mozilla.org/ci/ci-configuration/file/2458e282d4e38fe75c237d47d2ab1332b3a0676d/grants.yml#l1813)

Reply:
aha :)
In the future we probably want to standardize our naming, but this works, ty.